package/toltec-base/package

```diff
@@ -31,20 +31,23 @@ configure() {
         /opt/etc/profile
     echo "Disabling automatic update"
     disable-unit update-engine.service
-    if [[ "$arch" == "rm1" ]] && ! is-masked sys-subsystem-net-devices-usb1.device; then
+    if [[ "$arch" == "rm1" ]]; then
         echo "Disabling usb1 network device to avoid long boots"
-        systemctl mask sys-subsystem-net-devices-usb1.device
-    elif [[ "$arch" == "rm2" ]] && is-masked sys-subsystem-net-devices-usb1.device; then
+        ! systemctl is-masked sys-subsystem-net-devices-usb1.device && systemctl mask sys-subsystem-net-devices-usb1.device
+        ! systemctl is-masked busybox-ifplugd@usb1.service && systemctl mask busybox-ifplugd@usb1.service
+    elif [[ "$arch" == "rm2" ]]; then
         echo "Enabling usb1 network device to ensure usb SSH works"
-        systemctl unmask sys-subsystem-net-devices-usb1.device
+        systemctl is-masked sys-subsystem-net-devices-usb1.device && systemctl unmask sys-subsystem-net-devices-usb1.device
+        systemctl is-masked busybox-ifplugd@usb1.service && systemctl unmask busybox-ifplugd@usb1.service
     fi
 }
 
 postremove() {
-    if is-masked sys-subsystem-net-devices-usb1.device; then
+    if systemctl is-masked sys-subsystem-net-devices-usb1.device; then
         systemctl unmask sys-subsystem-net-devices-usb1.device
+        systemctl unmask busybox-ifplugd@usb1.service
     fi
-    if ! is-enabled "update-engine.service"; then
+    if ! systemctl is-enabled update-engine.service; then
```

Comment:
```suggestion
    if ! is-enabled update-engine.service; then
```
[install-lib contains an `is-enabled` method](https://github.com/toltec-dev/toltec/blob/f97179cf68861222b99a3321b13ae9c61c586d8a/scripts/install-lib#L22-L24) which properly hides output and only returns true if the unit is in an enabled state. This is why it was used previously, and not `systemctl` directly.